src/components/tl/mlx5/tl_mlx5.c

```diff
@@ -67,6 +67,27 @@ static ucc_config_field_t ucc_tl_mlx5_lib_config_table[] = {
      ucc_offsetof(ucc_tl_mlx5_lib_config_t, qp_conf.qp_max_atomic),
      UCC_CONFIG_TYPE_UINT},
 
+    {"MCAST_SX_DEPTH", "512", "Send context depth of the Mcast comm",
+     ucc_offsetof(ucc_tl_mlx5_lib_config_t, mcast_conf.sx_depth),
+     UCC_CONFIG_TYPE_INT},
+
+    {"MCAST_SX_INLINE", "128", "Minimal size for inline data send in Mcast",
```

Comment:
is it in bytes? if yes please use corresponding config type

Reply:
yes it is the number of the bytes, can you please clarify the suggestion?

Reply:
we use UCC_CONFIG_TYPE_MEMUNITS for such config variables